vllm/v1/kv_cache_interface.py

```diff
@@ -223,6 +223,14 @@ def max_memory_usage_bytes(self, vllm_config: VllmConfig) -> int:
         return self.page_size_bytes
 
 
+@dataclass
+class ShortConvSpec(MambaSpec):
```

Comment:
I prefer the `MambaSpec.mamba_type` approach in https://github.com/vllm-project/vllm/pull/21249. Can you do it in your PR so that we don't need to wait for that PR.